tests/conftest.py

```diff
@@ -338,96 +256,95 @@ def funding_nodes(private_key):
+    shutil.copy(PROTOCOL_CONFIG_FILE, TEST_PROTOCOL_CONFIG_FILE)
+    mirror_contract_data(TEST_PROTOCOL_CONFIG_FILE, DEPLOYMENTS_SUMMARY_FILE, NETWORK1, NETWORK1)
+    mirror_contract_data(TEST_PROTOCOL_CONFIG_FILE, DEPLOYMENTS_SUMMARY_FILE, NETWORK1, NETWORK2)
 
-    logging.info("Reuse pre-generated identities")
-    reuse_pregenerated_identities()
-
-    nodes = NODES.copy()
+    # SETUP NODES USING STORED IDENTITIES
+    logging.info("Reuse pre-generated identities and configs")
+    copy_identities()
 
+    # CREATE LOCAL SAFES AND MODULES FOR ALL THE IDS
     logging.info("Create safe and modules for all the ids, store them in args files")
-    nodes = create_local_safes(nodes, private_key)
 
-    try:
-        for node_id, node_args in nodes.items():
-            (proc, api) = setup_node(node_args)
-            nodes[node_id]["proc"] = proc
-            nodes[node_id]["api"] = api
-
-        logging.info(f"Wait for {len(nodes)} nodes to start up")
-
-        for node_id, node_args in nodes.items():
-            while True:
-                with open(f"{node_args['dir']}.log", "r") as f:
-                    logs = f.read()
-                    if "still unfunded, " in logs or "node is funded" in logs:
-                        logging.info(f"Node {node_id} up")
-                        break
-
-                sleep(0.1)
-
-        logging.info("Funding nodes")
-        funding_nodes(private_key)
-
-        logging.info("Collect node information")
-        for node_id, node_args in nodes.items():
-            (peer_id, address) = collect_node_information(node_args)
-            nodes[node_id]["peer_id"] = peer_id
-            nodes[node_id]["address"] = address
-
-        logging.info("Node setup finished, waiting 10 seconds before proceeding")
-        sleep(10)
-
-        yield nodes
-    except Exception as e:
-        logging.error(f"Creating a 7 node cluster - FAILED: {e}")
-    finally:
-        logging.debug("Tearing down the 7 node cluster")
-        for node_id, node_args in nodes.items():
-            cleanup_node(node_args)
-        pass
+    nodes: dict[str, Node] = deepcopy(NODES)
+
+    safe_custom_env: dict = {
+        "ETHERSCAN_API_KEY": "anykey",
+        "IDENTITY_PASSWORD": PASSWORD,
+        "DEPLOYER_PRIVATE_KEY": private_key,
+        "PRIVATE_KEY": private_key,
+        "PATH": PWD.parent.joinpath(f".foundry/bin:{os.environ['PATH']}").as_posix(),
+    }
+
+    for node in nodes.values():
+        logging.info(f"Creating safe and module for {node}")
+        node.create_local_safe(safe_custom_env)
+
+    for node in nodes.values():
+        logging.info(f"Setting up {node}")
+        node.setup_node(PASSWORD, TEST_PROTOCOL_CONFIG_FILE, PWD.parent)
+
+    # WAIT FOR NODES TO BE UP
+    logging.info(f"Wait for {len(nodes)} nodes to start up")
+
+    # minimal wait to ensure api is ready for `stardedz` call.
+    # This prevents cloging the log with "WARNING"s and "ERROR"s from the api
+    await asyncio.sleep(10)
```

Comment:
`startedz`should not generate the errors, it is meant to be polled even when the API is not running.

Reply:
Then something is not working with startedz. I had a lot of errors without the wait here.

Reply:
The status code is not accessible when using a generated API. At least nothing in the doc and researchs I did on SwaggerCodeGen shows a way to access the status code. THe response from a call when the API is not up is `None`. No field accessible, as it is a `None-type`.